src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -21,23 +21,21 @@ define([
 
         SHOW_GHOST_PAGES_KEY = 'column-navigation-show-ghost-pages',
 
-        /**
-         * node type constant for content
-         * @type {number}
-         */
-        TYPE_CONTENT = 1,
```

Comment:
this variable is used a few times in this file which results in that you cannot open a page form any more (: